### PR TITLE
Judicial System: Implements email sending

### PR DESCRIPTION
**Pull request by gudjong**
# Implements email sending

## What

Email is sent to prosecutors and judges with requests and rulings attached.

## Why

This is needed

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -16,8 +16,8 @@
 /apps/web*/                     	@island-is/kosmos-kaos @island-is/aranja
```
Comment (sindrig) [inline .github/CODEOWNERS:16]: Are all these indentation changes intended?

Comment (gudjong) [reply .github/CODEOWNERS:16]: Yes, they were. The file was kind of messy in that regard when I was adding my line to it. I noticed later that the file contains a mixture of tabs and spaces. Do we care?

Comment (sindrig) [reply .github/CODEOWNERS:16]: Ah I see it now. Yes, can you have it consistent - I guess we're using spaces in other places, can you replace all tabs with spaces while you're in there? :)

Comment (gudjong) [reply .github/CODEOWNERS:16]: Maybe there should be some auto formatting of this file.

Comment (sindrig) [reply .github/CODEOWNERS:16]: Agreed. https://github.com/island-is/island.is/issues/1259

**Review by oddsson — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (gudjong) [reply .github/CODEOWNERS]: Done replacing tabs.

**Review by oddsson — APPROVED** [review]

**Review by sindrig — APPROVED** [review]

**Pull request merged**